adafruit_epd/epd.py

```diff
@@ -11,14 +11,23 @@
 
 import time
 from micropython import const
-from digitalio import Direction
 from adafruit_epd import mcp_sram
 
+try:
+    """Needed for type annotations"""
+    from typing import Any, Union, Callable, Optional
+    from busio import SPI
+    from digitalio import Direction, DigitalInOut
+    from PIL.Image import Image
```

Comment:
For Image we have a Protocol over in our typing library for it so that we don't have to use the full PIL module as a dependency when it's only for typing. 

It can be imported like this and used instead of the PIL import:
```
from circuitpython_typing.pil import Image
```